src/components/main_page/Timeline.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+export default function Timeline(props) {
+    return (
+        <div className="timeline-container">
+            <div className="heading-container container timeline">
+                <div className="heading white">TIMELINE</div>
+                <div className="heading-line white"></div>
+                <div className="timeline-bar"></div>
+                <div className="timeline-content">
+                    <div className="timeline-event">
+                        <div className="event-name"><p>Event 01</p></div>
+                        <div className="circle-on-bar"></div>
+                        <div className="circle-event-connection"></div>
+                        <div className="event-picture"></div>
```

Comment:
Pictures were uploaded into Google drive. Try to add some into your code, see if they work well.